### ISSUE TITLE
Output of alphashape2poly.R?

### ISSUE THREAD

**Issue by MatthieuStigler** [babichmorrowc/hull2spatial#4]
HI

I don;t understand the output of `alphashape2poly.R`... help file says it's polygon, but it seems more to be a simple vector of points?

Thanks!

**Comment by babichmorrowc** [comment]
Good catch, thank you! I updated `ashape2poly` to take the vector to a SpatialPolygons object. Let me know if that works for you and if so I will close this issue

**Comment by MatthieuStigler** [comment]
Thanks for all the work! I had a few issues though:

- `Error: object 'graph_from_edgelist' not found` dependency to igraph not declared?
- the help page does not seem to be updated?
- the output sp seems ot be complaining about coordinates: _Deprecated Proj.4 representation: NA_ did you see the post about changes in proj 4?

Thanks!

**Comment by babichmorrowc** [comment]
Hi! Taking another look here:

- This should be fixed -- let me know if you are still having issues
- Which help page in particular?
- Can you provide a reproducible example for this error? I'm not getting this anywhere.
